specification/communication/data-plane/Rooms/preview/2022-02-01-preview/examples/Rooms_AddParticipants.json

```diff
@@ -0,0 +1,59 @@
+{
+  "parameters": {
+    "endpoint": "https://contoso.communication.azure.com/",
+    "api-version": "2022-02-01",
+    "roomId": "99199690362660524",
+    "addParticipantsRequest": {
+      "participants": [
+        {
+          "communicationIdentifier": {
+            "rawId": "8:acs:db75ed0c-e801-41a3-99a4-66a0a119a06c_00000010-ce28-064a-83fe-084822000666",
+            "communicationUser": {
+              "id": "8:acs:db75ed0c-e801-41a3-99a4-66a0a119a06c_00000010-ce28-064a-83fe-084822000666"
+            }
+          },
+          "role": "Presenter"
+        },
+        {
+          "communicationIdentifier": {
+            "rawId": "8:acs:db75ed0c-e801-41a3-99a4-66a0a119a06c_00000010-ce28-064a-83fe-084822000777",
+            "communicationUser": {
+              "id": "8:acs:db75ed0c-e801-41a3-99a4-66a0a119a06c_00000010-ce28-064a-83fe-084822000777"
```

Comment:
see my comment above https://github.com/Azure/azure-rest-api-specs/pull/19269/files#r897018392

Reply:
Please see me explanation at https://github.com/Azure/azure-rest-api-specs/pull/19269/files#r898249983
No changed required here. Aligned with Chat.

Reply:
I'd still prefer less verbosity in the examples and remove the `rawId` considering it's optional (similarly to [callingserver](https://github.com/Azure/azure-rest-api-specs/blob/ea1a21b30050e1a4b992dcb9d33db1245fa30b89/specification/communication/data-plane/CallingServer/preview/2021-08-30-preview/examples/CallConnections_AddParticipant.json)) and is just duplicating the `communicationUser.id`. Reasoning - it requires less typing and leaves less space for error for folks who use the REST API directly without an SDK.